modules/rest-api/pages/rest-initialize-node.adoc

```diff
@@ -74,7 +74,7 @@ curl -X POST \
 
 The sequence of tasks divided into _initialization_ and _provisioning_ is explained in xref:rest-api:rest-cluster-init-and-provisioning.adoc[Cluster Initialization and Provisioning].
 
-For each of the specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].
+For each of the specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Node], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].
```

Comment:
`For each of the specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Single Node], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].`